crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -709,7 +709,19 @@ impl CumulocityConverter {
         let command = command.to_owned();
         let payload = payload.to_string();
 
-        let mut logged = LoggedCommand::new(&command);
+        let mut args =
+            shell_words::split(&command).map_err(|e| CumulocityMapperError::ExecuteFailed {
+                error_message: e.to_string(),
+                command: command.to_string(),
+                operation_name: operation_name.to_string(),
+            })?;
+
+        let mut logged = LoggedCommand::new(args.remove(0));
```

Comment:
This line causes a panic if user didn't give anything in the custom command file since `args` is empty vector.

```
[exec]
topic = "c8y/s/ds"
on_message = "511"
command = ""
```